EIPS/eip-1193.md

```diff
@@ -303,6 +307,10 @@ class EthereumProvider extends EventEmitter {
     window.addEventListener('message', this._handleJsonrpcMessage.bind(this));
   }
 
+  static get isEIP1193() {
```

Comment:
The rest of the document describes this as non-static (i.e. line 109), which I think is better.

If you are dealing with an injected provider, checking a static field would be something like `window.ethereum.constructor.isEIP1193`. Checking a non-static field would be just `window.ethereum.isEIP1193`, which is cleaner.

That said, I have no strong preference. Just wanted to point out the inconsistency.